### ISSUE TITLE
Download won't go past 21 000 images on a single group

### ISSUE THREAD

**Issue by Timurama** [Bionus/imgbrd-grabber#2331]
I noticed this recently when I was trying to download every picture tagged with Hatsune Miku from gelbooru, which is a beefy number of 70 thousand so i guess it would have issues, my download stuck firm after twentieth batch, i know it's probably because of the search limit set on the site but is there any way to circumvent this? i also don't understand why the program needs to search the images in order to download them, why not, you know, download every image that comes under the specified tag that is located on the server? or is it not this simple?

**Comment by Timurama** [comment]
also what i found strange is that my download seem to went up to 20 999 while to my knowledge gelbooru permits only 1001 pages per search which is only 20 020 results if we're searching with defualt Grabber pagination, this limit somehow didn't stop the creation of the full twentieth batch, i wonder why is that, could the pictures be searched past 20 0020 or am i missing something?

**Comment by yami-no-tusbas** [comment]
yep, the solution is to use id:< and id:> on gelbooru, danbooru and affiliated booru like them. In your exemple the solution is to do like that : 
- we can see hatsune miku as 72970 results
- The problem is we don't know the id of each singular images, but we know the last id with the first result page (and we can eyeball it too) in this exemple it is id:6022756
- so now the best way would be to look for the images betweens "id" plages. We idealy want query close to 20k result, more will make us skip some image, so we start with "hatsune_miku id:<990000" since this get us 19876 results, close enought to 20k, frome then we add this query to the batch and proceed to the next one : "hatsune_miku id:>990000 id:<1900000" is 18363, and we continue until we get over the last id.

So the requests for all the hatsune_miku images would be : 

- hatsune_miku id:<990000 (19876)
- hatsune_miku id:>990000 id:<1900000 (18363)
- hatsune_miku id:>1900000 id:<4350000 (19803)
- hatsune_miku id:>4350000 id:<7000000 (14064)
Grand total : 72.106, so we miss a little but more likely images removed from the website, the id persist, but they got removed and are not counted in the results.

Save them in a path like : "hatsune_miku\%artist% - %id% - %rating% - %md5%.%ext%" or wathever you like, but don't use %search% unless you want a folder for each query.
